src/util.ts

```diff
@@ -39,33 +39,37 @@ export interface IPollingOptions {
  * @param {String} uri
  * @param {Function} isPollingDone
  * @param {Object} options for polling (maxAttempts, pollStep)
+ * @param {Object} settings for xhr request (method)
  * @private
  */
 export const handlePolling = (
     xhrGet: any,
     uri: string,
     isPollingDone: Function,
-    options: IPollingOptions = {}
+    options: IPollingOptions = {},
+    settings: any = {}
```

Comment:
this looks suspicious... what is the purpose of this change?

Reply:
well then I would say we should change the first argument and not add a settings like this :)